core/src/main/resources/hudson/search/Search/search-failed.jelly

```diff
@@ -43,13 +43,13 @@ THE SOFTWARE.
           <ol>
             <j:forEach var="i" items="${items}">
               <li id="item_${i.path}">
-                <a href="?q=${i.path}">${i.path}</a>
+                <a href="?q=${it.encodeQuery(i.path)}">${i.path}</a>
```

Comment:
The problem is creating search results that need encoding without depending on plugins